test/checkbox.test.tsx

```diff
@@ -1,81 +1,63 @@
+      .child("test `check` action", (test) => test.step(checkbox.check()).assertion(checkbox.is({ checked: true })))
+      .child("test `toggle` action", (test) =>
+        test
+          .step(checkbox.toggle())
+          .assertion(checkbox.is({ checked: true }))
+          .child("toggle twice", (test) => test.step(checkbox.toggle()).assertion(checkbox.is({ checked: false })))
+      )
   )
   .child("test `filter` by indeterminate", (test) =>
+    test.step(renderComponent({ indeterminate: true })).assertion(checkbox.is({ indeterminate: true }))
+  )
+  .child("test `filter` by checked", (test) =>
     test
-      .step("render", () => render(<FormControlLabel label="indeterminate" control={<Component indeterminate />} />))
-      .assertion(CheckBox({ indeterminate: true }).exists())
+      .step(render(<FormControlLabel label="checkbox" checked control={<Component />} />))
+      .assertion(checkbox.is({ checked: true }))
   )
   .child("test `filter` by disabled", (test) =>
     test
-      .step("render", () => render(<FormControlLabel label="disabled" disabled control={<Component />} />))
-      .assertion(CheckBox({ disabled: true }).exists())
+      .step(render(<FormControlLabel label="checkbox" disabled control={<Component />} />))
+      .assertion(Checkbox({ disabled: true }).exists())
   )
   .child("test `filter` by visible", (test) =>
-    test.step("render", () => render(<Component />)).assertion(CheckBox({ visible: false }).exists())
-  )
-  .child("test `click` action", (test) =>
-    test
-      .step("render", () => render(<FormControlLabel label="checkbox" control={<Component />} />))
-      .assertion(CheckBox().is({ checked: false, focused: false }))
-      .child("click and assert", (test) =>
-        test.step(CheckBox().click()).assertion(CheckBox().is({ checked: true, focused: true }))
-      )
-  )
-  .child("test `focus` action", (test) =>
-    test
-      .step("render", () => render(<FormControlLabel label="checkbox" control={<Component />} />))
-      .assertion(CheckBox().is({ focused: false }))
-      .child("focus and assert", (test) => test.step(CheckBox().focus()).assertion(CheckBox().is({ focused: true })))
-  )
-  .child("test `blur` action", (test) =>
-    test
-      .step("render", () => render(<FormControlLabel label="checkbox" control={<Component autoFocus />} />))
-      .assertion(CheckBox().is({ focused: true }))
-      .child("blur and assert", (test) => test.step(CheckBox().blur()).assertion(CheckBox().is({ focused: false })))
-  )
-  .child("test `check` action", (test) =>
-    test
-      .step("render", () => render(<FormControlLabel label="checkbox" control={<Component />} />))
-      .assertion(CheckBox().is({ checked: false }))
-      .child("check and assert", (test) => test.step(CheckBox().check()).assertion(CheckBox().is({ checked: true })))
+    test.step(render(<Component />)).assertion(Checkbox({ visible: false }).exists())
```

Comment:
let's be consistent about always putting `step()` and `assertion()` on different lines.

Reply:
This is how prettier works. I can add test files in `.prettierignore` or manually add a comment to disable exact lines.
Another solution is to restrict line width to 60 characters.

Reply:
Maybe it would be better to disable prettier only for tests? I don't want to format all other files manually.